spokestack/asr/spokestack/keyword_recognizer.py

```diff
@@ -0,0 +1,195 @@
+"""
+This module contains the Spokestack KeywordRecognizer which identifies multiple keywords
+from an audio stream.
+"""
+import logging
+import os
+from typing import List
+
+import numpy as np  # type: ignore
+
+from spokestack.context import SpeechContext
+from spokestack.models.tensorflow import TFLiteModel
+from spokestack.wakeword.ring_buffer import RingBuffer
```

Comment:
it would be better to pull the ring buffer module up to the root spokestack package, so you don't have dependences on the wakeword package